Restful booker.postman_collection.json

```diff
@@ -0,0 +1,236 @@
+{
+	"info": {
+		"_postman_id": "f7ab6d7d-30c5-4654-8d7b-82a10ee5eafe",
+		"name": "Restful booker",
+		"schema": "https://schema.getpostman.com/json/collection/v2.0.0/collection.json",
+		"_exporter_id": "24008468",
+		"_collection_link": "https://galactic-space-608894.postman.co/workspace/QAMP2023~1a5a43d1-7498-46bb-a827-52082927168a/collection/24008468-f7ab6d7d-30c5-4654-8d7b-82a10ee5eafe?action=share&creator=24008468&source=collection_link"
+	},
+	"item": [
+		{
+			"name": "Auth - CreateToken",
+			"request": {
+				"method": "POST",
+				"header": [
+					{
+						"key": "Content-Type",
+						"value": " application/json",
+						"type": "text"
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": "{\r\n    \"username\" : \"admin\",\r\n    \"password\" : \"password123\"\r\n}"
```

Comment:
Not even these credentials should be provided it is a dummy (valid) data but still convention should be followed.